crypto/conf/conf_sap.c

```diff
@@ -66,6 +66,8 @@ int ossl_config_int(const OPENSSL_INIT_SETTINGS *settings)
 
 #ifndef OPENSSL_SYS_UEFI
     ret = CONF_modules_load_file(filename, appname, flags);
+#else
+    ret = 1;
 #endif
```

Comment:
This might read better as

```
#ifdef OPENSSL_SYS_UEFI
    ret = 1;
#else
    ret = CONF_modules_load_file(filename, appname, flags);
#endif
```

Reply:
(I will add my approval, but if you want to update this, happen to re-review then)

Reply:
I prefer the common code being first.